js/client-topbar.js

```diff
@@ -5,6 +5,7 @@
 			'click a': 'click',
 			'click .username': 'clickUsername',
 			'click button': 'dispatchClickButton',
+			'dblclick .volume': 'toggleMute',
```

Comment:
`dblclick button[name=openSounds]`

You don't need the `.volume` class.

Reply:
Sorry, I'm unfamiliar with Backbone's View system (I had to look it up just to find out what it was doing) so I resorted to blithely cutting and pasting...